crates/bin/prove_block/src/state_utils.rs

```diff
@@ -1,153 +1,26 @@
 use std::collections::{HashMap, HashSet};
 
-use anyhow::Result;
 use cairo_vm::Felt252;
-use num_bigint::BigUint;
 use starknet::core::types::{BlockId, MaybePendingStateUpdate, StateDiff};
 use starknet::providers::jsonrpc::HttpTransport;
 use starknet::providers::{JsonRpcClient, Provider};
-use starknet_os::crypto::pedersen::PedersenHash;
 use starknet_os::io::InternalTransaction;
-use starknet_os::starknet::business_logic::fact_state::contract_class_objects::{
-    get_ffc_for_contract_class_facts, ContractClassLeaf,
-};
-use starknet_os::starknet::business_logic::fact_state::contract_state_objects::ContractState;
-use starknet_os::starknet::business_logic::fact_state::state::SharedState;
-use starknet_os::starkware_utils::commitment_tree::base_types::TreeIndex;
-// trait for Patricia Tree
-use starknet_os::starkware_utils::commitment_tree::binary_fact_tree::BinaryFactTree;
-use starknet_os::starkware_utils::commitment_tree::patricia_tree::patricia_tree::PatriciaTree;
-use starknet_os::storage::storage::FactFetchingContext;
 
-use crate::rpc_utils::{CachedRpcStorage, RpcStorage};
 use crate::utils::get_subcalled_contracts_from_tx_traces;
 
 #[derive(Clone)]
 pub struct FormattedStateUpdate {
     // TODO: Use more descriptive types
+    #[allow(unused)]
```

Comment:
These are pretty suspicious. If they're unused, why do we keep them?

Reply:
This is part of the TODOs (Im going to need to touch more stuff to remove them) 
But they are also gone in the next PR :P